server/src/MapChunk.ts

```diff
@@ -67,13 +68,60 @@ export class PlayerChunkDB extends BaseEntity implements PlayerChunk {
 	@JoinColumn({ name: 'hash' })
 	data!: ChunkData
 
+	/**
+	 * Saves a chunk
+	 * @param map_chunk chunk
+	 */
 	static async store(map_chunk: PlayerChunk) {
 		// TODO if PlayerChunk exists, and holds last reference to old hash, delete ChunkData at old hash
-		await ChunkDataDB.upsert(map_chunk.data, {
-			conflictPaths: ChunkDataDB.primaryCols,
-			skipUpdateIfNoValuesChanged: true,
+
+		// create object to store all data needed for chunk data
+		const chunkData = {
+			data: map_chunk.data.data,
+			hash: map_chunk.data.hash,
+			version: map_chunk.data.version,
+			chunkX: map_chunk.chunk_x,
+			chunkZ: map_chunk.chunk_z,
+			timestamp: map_chunk.ts,
```

Comment:
If we store the timestamp in seconds in the DB, it needs to be converted (multiply/divide by 1000) here and in other places.